### PR TITLE
Sanity Integration for Yoga Classes data in Index Page

### PR DESCRIPTION
**Pull request by anthopark**
Yoga class data can be fetched from sanity studio with `getStaticProps` on the index page.
- created sanity client
- query for yoga classes

### CODE REVIEW

**Comment by anthopark** [comment]
[Integrate FE to sanity for Index page](https://trello.com/c/eUgogwhy/9-integrate-fe-to-sanity-for-index-page)

File: nextjs-client/pages/index.js [hunk 1]
```diff
@@ -11,5 +13,18 @@ export default function Home() {
...
 
 Home.getLayout = function getLayout(page) {
-  return <Layout currentPage="home">{page}</Layout>;
+  return <Layout currentPage='home'>{page}</Layout>;
 };
+
+export async function getStaticProps() {
+  const yogaClassData = await client.fetch(yogaClassesQuery);
+
+  const data = { yogaClassData };
```
Comment (emilyPhee) [inline nextjs-client/pages/index.js:22]: `data = {yogaClassData}` data here is the same as the Home page received as props? Wasn't sure about the data that the Home page's prop `data`.

File: nextjs-client/pages/index.js [hunk 2]
```diff
@@ -11,5 +13,18 @@ export default function Home() {
...
+
+  return {
+    props: {
+      data,
```
Comment (emilyPhee) [inline nextjs-client/pages/index.js:26]: `data` props will be accessible in the same file?

**Review by emilyPhee — COMMENTED** [review]
great work @anthopark with the Sanity client. I left a few questions, thank you.

File: nextjs-client/pages/index.js (hunk 1 above)
Comment (anthopark) [reply nextjs-client/pages/index.js:22]: correct. you will access it with `props.data.yogaClassData` on the index page.

**Review by emilyPhee — APPROVED** [review]

**Pull request merged**